### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: scripts/upgrade [hunk 1]
```diff
@@ -49,7 +63,6 @@ fi
 if [ -z "$language" ]; then
 	language="en"
 	ynh_app_setting_set --app=$app --key=language --value=$language
-	ynh_app_setting_delete --app=$app --key=$lang
```
Comment (yalh76) [inline scripts/upgrade]: you should keep that, for removal from old versions

**Review by yalh76 — CHANGES_REQUESTED** [review]
LGTM

**Comment by ericgaspar** [comment]
!testme

File: scripts/upgrade [hunk 2]
```diff
@@ -59,6 +59,11 @@ if [ -z "$final_path" ]; then
 	ynh_app_setting_set --app=$app --key=final_path --value=$final_path
 fi
 
+# If lang exist, remove it
+if [ -v "$lang" ]; then
+	ynh_app_setting_delete --app=$app --key=$lang
+fi
```
Comment (yalh76) [inline scripts/upgrade]: won't work because `lang` is not loaded at the LOAD SETTING step, you should let it like it was

**Review by yalh76 — CHANGES_REQUESTED** [review]

**Comment by ericgaspar** [comment]
!testme

**Review by yalh76 — APPROVED** [review]

**Pull request merged**